internal/ircserver/cmd_privmsg.go

```diff
@@ -59,6 +59,21 @@ func (i *IRCServer) cmdPrivmsg(s *Session, reply *Replyctx, msg *irc.Message) {
 			Params:  []string{msg.Params[0], msg.Trailing()},
 		})
 		return
+	} else if strings.HasPrefix(msg.Params[0], "$") {
+		if msg.Params[0] == "$"+i.ServerPrefix.Name || msg.Params[0] == "$*" {
```

Comment:
Given that RobustIRC inherently does not support server-to-server communication, there can only ever be one IRC server in a RobustIRC network. 

Instead of checking that the parameter matches i.ServerPrefix.Name, we could just treat all targets that start with `$` as a network-global message and remove this validation/error message. That would make the command a little more user-friendly.